maven.gradle

```diff
@@ -0,0 +1,35 @@
+rootProject.allprojects {
+    String rosMavenPath = System.getenv("ROS_MAVEN_PATH")
+    String rosMavenRepository = System.getenv("ROS_MAVEN_REPOSITORY")
+    repositories {
+        if (rosMavenPath != null) {
+            rosMavenPath.tokenize(":").each { path ->
+                //noinspection GroovyAssignabilityCheck
+                maven {
+                    // We can't use uri() here because we aren't running inside something
+                    // that implements the Script interface.
+                    url "file:${path}"
+                }
+            }
+        }
+        //noinspection GroovyAssignabilityCheck
+        maven {
+            url "http://repository.springsource.com/maven/bundles/release"
+        }
+        //noinspection GroovyAssignabilityCheck
+        maven {
+            url "http://repository.springsource.com/maven/bundles/external"
+        }
+        if (rosMavenRepository != null) {
+            //noinspection GroovyAssignabilityCheck
+            maven {
+                url rosMavenRepository
+            }
+        }
+        //noinspection GroovyAssignabilityCheck
+        maven {
+            url "https://github.com/rosjava/rosjava_mvn_repo/raw/master"
+        }
+        mavenCentral()
+    }
+}
```

Comment:
I'd like to get rid of this so we only have one source of maven repos in the [ros plugin](https://github.com/rosjava/rosjava_bootstrap/blob/indigo/gradle_plugins/src/main/groovy/org/ros/gradle_plugins/RosPlugin.groovy#L31).

@talregev says this doesn't work on windows. Started an issue in #46.

Will leave this `maven.gradle` file in until we resolve the plugin problem.